torch/csrc/api/include/torch/nn/options/dropout.h

```diff
@@ -4,16 +4,18 @@
 #include <torch/csrc/WindowsTorchApiMacro.h>
 #include <torch/types.h>
 
+
 namespace torch {
 namespace nn {
 
 /// Options for `Dropout` and `FeatureDropout`.
 struct TORCH_API DropoutOptions {
-  /* implicit */ DropoutOptions(double rate = 0.5);
+  /* implicit */ DropoutOptions(double p = 0.5, bool inplace = false);
```

Comment:
I think we can put the default value for `inplace` as part of the `TORCH_ARG` declaration instead, something like
```cpp
TORCH_ARG(bool, inplace) = false;
```
instead of specifying it as default value to the constructor argument. (Ideally we should do the same for `p` as well, but there are probably too many people using `Dropout(0.5)` directly, and we have to preserve it for better backward compatibility.)